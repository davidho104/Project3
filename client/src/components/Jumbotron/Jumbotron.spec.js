import React from 'react';
import { shallow } from 'enzyme';
import Jumbotron from './Jumbotron';

describe('Jumbotron', () => {
  let wrapper;
  beforeEach(() => wrapper = shallow(<Jumbotron />));

  it('should render correctly', () => expect(wrapper).toMatchSnapshot());

  it('should render <div />', () => {
    expect(wrapper.find('div').length).toEqual(1);
  });

  // it('renders the value of displayValue', () => {
  //   wrapper.setProps({ displayValue: 'test' });
  //   expect(wrapper.text()).toEqual('test');;
  // });q
});