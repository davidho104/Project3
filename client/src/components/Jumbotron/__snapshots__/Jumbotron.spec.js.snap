// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jumbotron should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Jumbotron />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": undefined,
      "className": "jumbotron",
      "style": Object {
        "clear": "both",
        "height": 300,
        "paddingTop": 120,
        "textAlign": "center",
      },
    },
    "ref": null,
    "rendered": null,
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": undefined,
        "className": "jumbotron",
        "style": Object {
          "clear": "both",
          "height": 300,
          "paddingTop": 120,
          "textAlign": "center",
        },
      },
      "ref": null,
      "rendered": null,
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
